src/main/java/org/myrobotlab/service/Wikipedia.java

```diff
@@ -165,9 +163,19 @@ private SearchResults searchWikipedia(String searchText, Boolean publishText, Bo
       if (bytes != null) {
         String response = new String(bytes, StandardCharsets.UTF_8);
         @SuppressWarnings("unchecked")
-        LinkedHashTreeMap<String, Object> json = CodecUtils.fromJson(response, LinkedHashTreeMap.class);
+        LinkedHashMap<String, Object> json = CodecUtils.fromJson(response, LinkedHashMap.class);

```

Comment:
should we deserialize to Map?

Reply:
not sure if a default map container is defined .. I'll check

Reply:
verified worky with general Map

Reply:
updated